.github/workflows/tests.yml

```diff
@@ -141,7 +141,7 @@ jobs:
           if [ -n "$DEPENDENCIES" ]; then
             source .venv/bin/activate
             echo "Installing packages: $DEPENDENCIES"
-            uv pip install $DEPENDENCIES --system
+            uv pip install $DEPENDENCIES
```

Comment:
This just fixes a slight oversight from #11515. We've already created a virtual environment here (we were doing so before we switched to `uv`), so there's no need to use the `--system` flag here.